<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeatureStyleSettingDlg</class>
 <widget class="QDialog" name="FeatureStyleSettingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>162</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FeatureStyleSettingDlg</string>
  </property>
  <widget class="QComboBox" name="iconComboBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>32</y>
     <width>118</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>118</width>
     <height>45</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>132</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>选择标注属性：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>118</width>
     <height>45</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>132</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>图层高度：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>80</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>m</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="lyHeightTextEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>118</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
